RockLib.HealthChecks.AspNetCore/HealthCheckMiddlewareExtensions.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Hosting;
+using RockLib.Configuration;
```

Comment:
Ideally we don't want to add RockLib.Configuration to any projects. Its' last release was 3 months ago and it will not be updated anymore. We are currently removing it from all existing projects.

Reply:
Worth noting I did not introduce the package into the library.  Other classes were using it, and it seemed appropriate to remain consistent (one health check to the next).  If you want me to *not* use it (here), is there a recommended direction to move in?

Reply:
`RockLib.Configuration` wasn't actually used.  [Easy fix!](https://github.com/RockLib/RockLib.HealthChecks/pull/58/commits/0463d0edc001c0b5393e97f03e8f324067f2a629)